packages/web3-shared/evm/hooks/useERC721TokenDetailedCallback.ts

```diff
@@ -10,18 +14,39 @@ export function useERC721TokenDetailedCallback(contractDetailed: ERC721ContractD
     const erc721TokenContract = useERC721TokenContract(contractDetailed?.address ?? '')
     const erc721TokenDetailedCallback = useCallback(async () => {
         if (!erc721TokenContract || !contractDetailed || !tokenId) return
-        if (!GET_SINGLE_ASSET_URL)
-            return getERC721TokenDetailedFromChain(contractDetailed, erc721TokenContract, tokenId)
+        let _tokenDetailedFromChain: ERC721TokenDetailed | undefined
+        let tokenDetailedFromOpensea: ERC721TokenDetailed | null = null
```

Comment:
Plz, keep them the same style.

```suggestion
        let tokenDetailedFromChain: ERC721TokenDetailed | null = null
        let tokenDetailedFromOpensea: ERC721TokenDetailed | null = null
```